improver_tests/calibration/reliability_calibration/test_AggregateReliabilityCalibrationTables.py

```diff
@@ -33,225 +33,160 @@
 import unittest
```

Comment:
CI is just Flake8 complaining of this unused import

Reply:
Fixed in fb4c1a39